src/HealthCheck.Core/Extensions/TaskExtensions.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System;
 using System.Threading.Tasks;
 
-namespace HealthCheck.Core.Helpers
+namespace HealthCheck.Core.Extensions
```

Comment:
Unnecessary namespace name change.

Reply:
Having helpers it is like having something bad designed in codebase. I am trying to avoid helper classes and namespaces, use more strict definitions. But again, my ex team want to use your library so i will not insist.

Reply:
"Helpers" is just a name, I don't think it's indicative of bad design in itself.

Reply:
Well it s not just a name. It is a part of solution structure. It is something we are working with on daily basis and on which we are relying on. We want to look at collapsed folder name and to play quiz what is inside.

Reply:
I actually think "Helpers" is a better name than "Extensions" in this case; it's signalling that this subfolder contains utility components used internally by the "Core" components. Calling it "Extensions" would lead one to think that there are only extension methods inside, or that the components are extensions to "Core" itself (similar to the convention of naming NuGet "addon" packages to [main package].Extensions, for example `Nlog.Extensions`). "Utilities" would also be a good name, but I don't see the point in changing it.

Reply:
Helpers may be parent folder for Extensions folder but i strongly disagree on mixing up this and having such general parts of project. Again, if you so strongly disagree after this comment i will change it back